pgoapi/pgoapi.py

```diff
@@ -226,23 +227,62 @@ def heartbeat(self):
         self._heartbeat_number += 1
         return res
 
-    def walk_to(self, loc, waypoints=[]):  # location in floats of course...
+    def _walk(self, loc, waypoints=[], directly=False):  # location in floats of course...
         steps = get_route(self._posf, loc, self.config.get("USE_GOOGLE", False), self.config.get("GMAPS_API_KEY", ""),
                           self.experimental and self.spin_all_forts, waypoints)
         catch_attempt = 0
+        base_travel_link = "https://www.google.com/maps/dir/%s,%s/" % (self._posf[0], self._posf[1])
+        step_size = self.config.get("STEP_SIZE", 200)
```

Comment:
I guess that could be an issue where we try to jump 4km really quickly rather than traveling slowly on the route/steps generated from the googlemaps API.  In general I haven't seen that problem around my area and I've been running it all day as I improved on this code.

I'm not sure what a good solution to safeguard against that would be, I guess we could take *all* steps and measure the distances between each step and compare that as well.  So if the steps take 5km but jumping straight to it is 1km we don't do it, we continue to follow the steps until we are within 1km